File: rnmodules/react-native-kb/android/src/main/java/com/reactnativekb/DarkModePrefHelper.java
package com.reactnativekb;

public class DarkModePrefHelper {
  public static DarkModePreference fromString(String prefString) {
    switch (prefString) {
      case "alwaysDark":
        return DarkModePreference.AlwaysDark;
      case "alwaysLight":
        return DarkModePreference.AlwaysLight;
      default:
        return DarkModePreference.System;
    }
  }
}

File: rnmodules/react-native-kb/android/src/main/java/com/reactnativekb/DarkModePrefHelper.kt
package com.reactnativekb

object DarkModePrefHelper {
    fun fromString(prefString: String): DarkModePreference {
        return when (prefString) {
            "alwaysDark" -> DarkModePreference.AlwaysDark
            "alwaysLight" -> DarkModePreference.AlwaysLight
            else -> DarkModePreference.System
        }
    }
}
